web3/_utils/async_transactions.py

```diff
@@ -243,3 +247,39 @@ async def async_replace_transaction(
         async_w3, current_transaction, new_transaction
     )
     return await async_w3.eth.send_transaction(new_transaction)
+
+
+async def async_extract_valid_transaction_params(transaction_params: TxData) -> TxParams:
```

Comment:
I don't see any reason why this function needs to be asynchronous, so we can get rid of this and use `extract_valid_transaction_params` from `web3/_utils/transactions.py`

Reply:
Hello, thank you for reviewing. I can remove the async version of the extract_valid_transaction_params function. However, I am not certain why the CI is failing with it. It would be greatly appreciated if you could take care of it. Thanks!